sdk/ml/ci.yml

```diff
@@ -33,11 +33,6 @@ extends:
     ValidateFormatting: true
     TestTimeoutInMinutes: 75
     TestProxy: true
-    MatrixConfigs:
```

Comment:
How does removing this code enabled 3.12?

Also, is the commented info below related to this deleted code or something else?

Reply:
The current default is to include 3.12 in our test runs. In the PR that made this so, we received an exclusion so that we _wouldn't_ run on 3.12.

This PR undoes the commit that gave us the exclusion https://github.com/Azure/azure-sdk-for-python/pull/31758/commits/787cf8fa55943f1532e80a833e11c0e0c14dd9c2